Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -602,6 +613,16 @@ def create(
         if version not in self.projects[project]['versions']:
             raise ValueError("'{}' is not a recognized version for '{}'".format(version, project))
 
+        if keywords is not False:
```

Comment:
I don't think we should prompt here. Instead, `keywords` should be a list that the caller can provide, you should cross-check that list against `self.keywords()`, but you should only emit logging here if the caller provided keywords you can't add.

Reply:
That makes sense! Made the changes.